### ISSUE TITLE
[Active learning] Tutorial

### ISSUE THREAD

**Issue by philippmwirth** [lightly-ai/lightly#146]
# Tutorial for Active Learning

## Description
Add a simple tutorial on active learning to the [docs](https://docs.lightly.ai/). 

## Tasks

- [x] Add a tutorial about active learning for classification to the docs

This tuturial includes the following steps:
1. Define the prerequisites: a dataset as images on the disk and the corresponding labels. The validation set also has to be defined (or split in a later step). 
2. Create embeddings for the dataset. Decision: Using Code - another 100 lines? Using CLI?
3. Create a new dataset in the Webapp and copy the token and dataset_id.
4. Upload the `embedding.csv`
5. Define a classifier on your dataset with training, predict_probabilities and evaluation methods. It must support defining the labeled set as a list of filenames or an external function performing this split must be created.
6. (Optional, only needed if no validation set is defined): Split the dataset using the random sampler into two parts with a predefined ratio. One is used as the training set/query tag, the other one as the validation set. 
7. Perform an initial sampling (e.g. with coreset or random) on the training set and get the filenames of the labeled set back. 
8. Train the classifier on the labeled set.
9. Evaluate the classifier on the validation set and store the result somewhere.
10. Predict the probabilities of the classes with the classifer on the unlabeled set. 
11. Save the predictions on the unlabeled_set in the al_scorer
12. Peform a CORAL sampling with the al_scorer (internally uploads the scores) and get the filenames of the labeled set back.
13. Continue the loop starting from step 8 for a few iterations.
14. Plot the performance of the agent (saved in step 9) over multiple active learning steps.

## Acceptance Criteria

- [x] Tutorial works
- [x] Tutorial introduces all concepts which are new in lightly AL.
